### ISSUE TITLE
Untracked version 0.8

### ISSUE THREAD

**Issue by taldcroft** [sot/Ska.TelemArchive#1]
It appears there was a version 0.8 of Ska.TelemArchive that isn't in git version control (as far as I can see).  This update is going to get stomped by 0.7.1 that is based on 0.7.

I _think_ that this package isn't called by anything now, so just documenting this instead of fixing it.

```
skatest-kadi$ diff {Ska/TelemArchive/,/proj/sot/ska/arch/x86_64-linux_CentOS-6/lib/python2.7/site-packages/Ska.TelemArchive-0.8-py2.7.egg/Ska/TelemArchive/}fetch.py
10a11,12
> import sqlite3 as sqlite
> import numpy
119c121,125
<                 quality |= column.quality

---
>                 try:
>                     quality |= column.quality
>                 except TypeError:
>                     if not numpy.isnan(column.quality):
>                         raise
234d239
<         from pysqlite2 import dbapi2 as sqlite
```

**Comment by jeanconn** [comment]
Though not sure what you meant about "I think that this package isn't called by anything now," as I thought the package was still being called by the aiprops/observations code which I thought was still running.

**Comment by taldcroft** [comment]
Good point, but I think the only thing in that package still being used is Ska.TelemArchive.data_table, which is not different in 0.8:

```
kadi$ pwd
/home/aldcroft/git/telem_archive

# Here are the things that are running now
kadi$ grep exec task_schedule.cfg 
#  exec: Name of executable.  Can have $ENV{} vars which get interpolated.
#        If bin_dir is defined then bin_dir is prepended to non-absolute exec names.
# Note the syntax 'exec <number> : cmd', which means that the given command is
# executed only once for each <number> of times the task is executed.  In the
      exec get_products.pl
      exec make_aiprops.py --dbi sqlite
      exec make_aiprops.py --dbi sybase
      exec make_obs_aiprops.py --dbi sqlite
      exec make_obs_aiprops.py --dbi sybase
      exec make_obspar.py --dbi sqlite
      exec make_obspar.py --dbi sybase

# In those top level scripts, what gets imported?
kadi$ grep Ska.TelemArchive *.py
make_aiprops.py:from Ska.TelemArchive.data_table import DataTable
setup.py:setup(name='Ska.TelemArchive',
setup.py:      py_modules = ['Ska.TelemArchive.fetch',
setup.py:                    'Ska.TelemArchive.fetch_client',
setup.py:                    'Ska.TelemArchive.fetch_server',
setup.py:                    'Ska.TelemArchive.data_table'],
setup.py:      packages=['Ska', 'Ska.TelemArchive'],
```

But at this point it would probably have been faster to just put back this diff. Sigh.

**Comment by taldcroft** [comment]
Closed by 74011dd7.  (Note that commit messages got mixed up.)
